CHANGELOG.md

```diff
@@ -1,27 +1,222 @@
+- SQL statements with single-line comments unable to execute [#2085](https://github.com/oceanbase/odc/pull/2085)
+- Incorrect offsets for last SQL command without a trailing delimiter [#1970](https://github.com/oceanbase/odc/pull/1970)
+- Incompatibility of anonymous block execution module with Oracle 11g [#1759](https://github.com/oceanbase/odc/pull/1759)
+- DBMS output failed to output spaces [#1051](https://github.com/oceanbase/odc/issues/1970)
+- `#` and `$` character disappeared after SQL window formatted the code [#1490](https://github.com/oceanbase/odc/issues/1490)
+- Auto-complete in SQL window not available for MySQL datasource [#1718](https://github.com/oceanbase/odc/issues/1718)
+
+Result-set
+
+- Degrading performance when modifying multiple rows concurrently in result-set [#2007](https://github.com/oceanbase/odc/pull/2007)
+- In OceanBase MySQL mode, the datetime data type loses precision when displayed [#1996](https://github.com/oceanbase/odc/pull/1996)
+- Switching between viewing BLOB field text and hexadecimal images may cause the UI to freeze [#300](https://github.com/oceanbase/odc/issues/300)
 
+Table Object
 
+- Disparity in column names order between Index and Constraint views [#1948](https://github.com/oceanbase/odc/pull/1948)
+- Inability to view table details for MySQL version 5.6 [#1635](https://github.com/oceanbase/odc/pull/1635)
+- Sofa ODP table detail view not accessible [#2043](https://github.com/oceanbase/odc/pull/2043)
+- Unable to change NOT NULL fields to NULL in table structure editor [#1441](https://github.com/oceanbase/odc/issues/1441)
+- When partitioned table has multiple maximum values, only one is displayed [#1501](https://github.com/oceanbase/odc/issues/1501)
+- Button to delete table primary key is greyed out and not clickable [#1874](https://github.com/oceanbase/odc/issues/1874)
 
+Import/Export
+
+- SQL statement split incorrectly causing task failure when comments contain `;` [#417](https://github.com/oceanbase/odc/issues/417)
+- Export task fails when type name is in lowercase [#631](https://github.com/oceanbase/odc/issues/631)
+- Exporting trigger objects fails [#750](https://github.com/oceanbase/odc/issues/750)
+- Export task fails when function name contains special characters [#1331](https://github.com/oceanbase/odc/issues/1331)
+- During Oracle mode export of indexes, index names are prefixed with database name [#1491](https://github.com/oceanbase/odc/issues/1491)
+- `DELIMITER $$` separator sticks to table name when exporting procedure structure [#1746](https://github.com/oceanbase/odc/issues/1746)
+- After creating an export task, terminating it shows task status as executed successfully [#1752](https://github.com/oceanbase/odc/issues/1752)
+- When exporting packages, object type in task details doesn't show the package body [#1755](https://github.com/oceanbase/odc/issues/1755)
+- Import task fails when CSV file includes DATE type [#2079](https://github.com/oceanbase/odc/issues/2079)
```

Comment:
- Task exception caused by incorrect SQL statement splitting when comments contain `;`. [#417](https://github.com/oceanbase/odc/issues/417)
- Export task fails when type names are lowercase. [#631](https://github.com/oceanbase/odc/issues/631)
- Export failure when exporting trigger objects. [#750](https://github.com/oceanbase/odc/issues/750)
- Export task fails when function names contain special characters. [#1331](https://github.com/oceanbase/odc/issues/1331)
- When exporting indexes in Oracle mode, the index name is prefixed with the database name. [#1491](https://github.com/oceanbase/odc/issues/1491)
- When exporting the schema of stored procedures, the `DELIMITER $$` delimiter is concatenated with the table name. [#1746](https://github.com/oceanbase/odc/issues/1746)
- After creating an export task, terminating the export task will still display the task status as executed successfully. [#1752](https://github.com/oceanbase/odc/issues/1752)
- When exporting packages, the object type is not displayed as the package body in task details. [#1755](https://github.com/oceanbase/odc/issues/1755)
- Import fails when importing CSV files containing DATE types. [#2079](https://github.com/oceanbase/odc/issues/2079)